### PR TITLE
K8 experiment - fixes

### PR DESCRIPTION
**Pull request by pshemass**
Couple of fixes after first run on Kubernetes.

### CODE REVIEW

**Comment by mschuwalow** [comment]
Do we want to wait with this until @mijicd's PR is ready?

**Comment by mijicd** [comment]
Let's merge this, one more adjustment won't change things for me.

File: build.sbt [hunk 1]
```diff
@@ -63,9 +63,18 @@ lazy val keeper = project
     fork in test := true
   )
 
+lazy val dockerSettings = Seq(
+  dockerBaseImage := "openjdk:11",
```
Comment (mijicd) [inline build.sbt:67]: We can probably go with something thinner here, like one of those alpine jre images.

Comment (pshemass) [reply build.sbt:67]: sure

**Pull request merged**